aws/windows-agent/main.tf

```diff
@@ -35,19 +31,20 @@ module "dcos" {
   dcos_variant = "open"
 
   dcos_version              = "1.13.0"
-  ansible_bundled_container = "sebbrandt87/dcos-ansible-bundle:windows-support"
+  //TO DO : change a tag once a new build of docker image available
+  ansible_bundled_container = "mesosphere/dcos-ansible-bundle:feature-windows-support-039d79d"
 
   # provide a SHA512 hashed password, here "deleteme"
   dcos_superuser_password_hash = "$6$rounds=656000$YSvuFmasQDXheddh$TpYlCxNHF6PbsGkjlK99Pwxg7D0mgWJ.y0hE2JKoa61wHx.1wtxTAHVRHfsJU9zzHWDoE08wpdtToHimNR9FJ/"
   dcos_superuser_username      = "demo-super"
 
-  additional_windows_private_agent_ips       = ["${concat(module.winagent.private_ips)}"]
-  additional_windows_private_agent_passwords = ["${concat(module.winagent.windows_passwords)}"]
+  additional_windows_private_agent_ips       = ["${concat(module.windows-instance.private_ips)}"]
+  additional_windows_private_agent_passwords = ["${concat(module.windows-instance.windows_passwords)}"]
 }
 
-module "winagent" {
```

Comment:
the module name here is `winagent` and should stay like this entirely.
a module name does not need to match the source it is created from.